### PR TITLE
Jdbc Implementation for the Member model

### PR DESCRIPTION
**Pull request by JesterCharles**
Our first set of JDBC implementation for the Member model

### CODE REVIEW

File: Week-2/LibraryApp/pom.xml
Comment (EricTrainingRev) [inline Week-2/LibraryApp/pom.xml:1]: Design Requirements Documentation says not to include dependency location comments

Comment (JesterCharles) [reply Week-2/LibraryApp/pom.xml:1]: Well that's tooo bad, it's going in there

File: Week-2/LibraryApp/src/main/java/com/revature/LibraryApp/daos/MemberDao.java
Comment (EricTrainingRev) [inline Week-2/LibraryApp/src/main/java/com/revature/LibraryApp/daos/MemberDao.java:1]: LGTM

**Review by EricTrainingRev — APPROVED** [review]
LGTM

**Review by EricTrainingRev — APPROVED** [review]

**Pull request merged**